pandas/tests/indexing/test_categorical.py

```diff
@@ -64,11 +68,28 @@ def test_loc_scalar(self):
         df2.loc["d"] = 10
         tm.assert_frame_equal(df2, expected)
 
-        msg = "'fill_value=d' is not present in this Categorical's categories"
-        with pytest.raises(TypeError, match=msg):
-            df.loc["d", "A"] = 10
-        with pytest.raises(TypeError, match=msg):
-            df.loc["d", "C"] = 10
+        # Setting-with-expansion with a new key "d" that is not among caegories
```

Comment:
might be worth splitting this test (e.g. errors to a new tests) and even these cases